e2e/pages/Home/HomePage.ts

```diff
@@ -1,37 +1,39 @@
-import { expect } from '@playwright/test';
+import { Locator, expect } from '@playwright/test';
 import { Page } from 'playwright';
 
 class HomePage {
-  page: Page;
-  activeProgramsBanner = 'p.ion-no-margin';
-  programCard = 'app-program-card';
-  openPAsForRegistrationButton = 'ion-button.ion-margin-start';
-  okButton = 'ion-button[fill="solid"][color="primary"]:has-text("OK")';
+  readonly page: Page;
+  readonly activeProgramsBanner: Locator;
+  readonly programCard: Locator;
+  readonly openPAsForRegistrationButton: Locator;
+  readonly okButton: Locator;
 
   constructor(page: Page) {
     this.page = page;
+    this.activeProgramsBanner = this.page.locator('p.ion-no-margin');
+    this.programCard = this.page.locator('[data-testid="program-card"]');
```

Comment:
Can you use the verbose locators wherever applicable? It makes it easier to read, and also makes it easier to prevent switching to other kinds of selectors by mistake.

```suggestion
    this.programCard = this.page.getByTestId('program-card');
```

Reply:
Done but left this part untouched as Sun is working on it at the moment so I'll give her feedback based on your suggestion